docs/source/conf.py

```diff
@@ -209,6 +209,7 @@ def get_description(rst_file_name):
     "geopm_version.3",
     "geopmwrite.1",
     "geopmaccess.1",
+    "geopmexporter.1",
```

Comment:
Not sure what order these are in, nor why this is in the middle and not at the end.  It's almost like someone tried to alphabetize them until ``geopmaccess.1`` was added?  Unclear.